confluence-syntax-xhtml/src/test/resources/confluence+xhtml10/specific/task/task1.test

```diff
@@ -18,28 +18,6 @@
 .expect|event/1.0
 .#-----------------------------------------------------
 beginDocument
-beginList [BULLETED]
-beginListItem
-onWord [(((9745)))]
-onSpace
-onWord [First]
-onSpace
-onWord [task]
-onSpace
-onWord [completed]
-onSpace
-onWord [on]
-onSpace
-onMacroInline [time] [datetime=2020-01-01]
-endListItem
-endList [BULLETED]
-beginList [BULLETED]
-beginListItem
-onWord [(((9744)))]
-onSpace
-onWord [Second]
-onSpace
-onWord [task]
-endListItem
-endList [BULLETED]
+onMacroStandalone [task] [id=1|status=complete] [First task completed on ]
```

Comment:
Looks like the time is lost.

Reply:
I **think** it has something to do with how the tests are run (with `RenderingTestSuite.class`) and therefore it is expected in this case. I followed the example of the Layout macros (layout, layout-cell, layout-section) and added another test that will be run with the `FilterTestSuite.class` which has the expected result.

Disclaimer: I don't fully understand how the Tests work and how running them with different TestSuites is supposed to affect them.

Reply:
With the changes to the `<ac:task-list>` tag, I think it would be best to remove this test and leave only the one that is being run with the `FilterTestSuite`.

Reply:
> Disclaimer: I don't fully understand how the Tests work and how running them with different TestSuites is supposed to affect them.

It's not so much the test framework than the way `AbstractRichContentTagHandler` behave depending on the entry point used (as input filter or as rendering parser). Anyway, it's out of context here, and it's probably simpler to just get rid of that missleading test.